snuba/clickhouse/native.py

```diff
@@ -260,6 +260,9 @@ def execute(
                         time.sleep(sleep_interval_seconds)
                         continue
 
+                    if e.code == errors.ErrorCodes.MEMORY_LIMIT_EXCEEDED:
+                        pass
```

Comment:
what does this do?

Reply:
Accidentally committed that, deleted already.